packages/button-react/documentation/buttons.mdx

```diff
@@ -51,3 +52,11 @@ Knappetekster skal være så enkle og korte som mulig og skal oppfordre til hand
 <PrimaryButton className="jkl-layout-spacing--medium-right">Send inn</PrimaryButton>
 <SecondaryButton className="jkl-layout-spacing--medium-right">Lagre</SecondaryButton>
 <TertiaryButton className="jkl-layout-spacing--medium-right">Avbryt</TertiaryButton>
+
+## Knapper som er lenker
+
+Avhengig av situasjonen kan det hende en knapp teknisk sett er et `<a />`-element. Jøkul støtter å ha knappeklasser på `<a />`.
+
+<ComponentExample component={Anchors}>
```

Comment:
Litt tilfeldig at jeg kom på dette nå, men en feature som vi bruker altfor lite (kanskje ikke i det hele tatt?) er muligheten for å sette tittel på komponenteksemplene. Det er den lille skriften oppe i hjørnet av eksemplet som vanligvis er "Komponent". Kanskje vi kunne skrevet "Lenker stilet som knapper"" eller noe tilsvarende her?

Trenger sikkert ikke ta det i denne PRen siden du har styret så mye allerede, men er kanskje noe vi burde tenke på, både her og andre steder i dokumentasjonen? Kanskje også skrive "PrimaryButton", "SecondaryButton" og "TertiaryButton" i komponenteksemplene øverst f.eks.?